packages/components/src/LightBox/LightBox.mdx

```diff
@@ -0,0 +1,73 @@
+---
+name: LightBox
+menu: Components
+route: /components/light-box
+---
+
+import { useState } from "react";
+import { Playground, Props } from "docz";
+import { ComponentStatus } from "@jobber/docx";
+import { Button } from "../Button";
+import { LightBox } from ".";
+
+# Light Box
+
+<ComponentStatus stage="pre" responsive="no" accessible="no" />
+
+LightBox is a component designed to display an array of images to the user. A
+user clicks on a control and is shown a carousel style presentation of the
+defined images. This component is a wrapper of the frontend-collective's
```

Comment:
:"This component is a wrapper of..." sentence could be its' own line, it doesn't feel conceptually related to the other sentences in this paragraph. Should we link out to the frontend-collective as well?